python/examples/panorama_sfm.py

```diff
@@ -20,6 +22,8 @@
 import pycolmap
 from pycolmap import logging
 
+NDArrayNx2 = Annotated[npt.NDArray[np.float64], Literal[Never, 2]]
```

Comment:
`Never` usually means unreachable code, and is introduced only in 3.11. Better use string litterals:
```suggestion
NDArrayNx2 = Annotated[npt.NDArray[np.float64], Literal['N', 2]]
```

Reply:
Somehow this didn't work with 3.9 and now also not with 3.10. I found another way.